test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/ContractMintHTSSuite.java

```diff
@@ -377,9 +375,23 @@ private HapiApiSpec transferNftAfterNestedMint() {
 				).then(
 						getTxnRecord(nestedTransferTxn).andAllChildRecords().logged(),
 						childRecordsCheck(nestedTransferTxn, SUCCESS,
-								recordWith().status(SUCCESS),
-								recordWith().status(SUCCESS).tokenTransfers(NonFungibleTransfers.changingNFTBalances().
-												including(nonFungibleToken, TOKEN_TREASURY, theRecipient, 1)))
+							recordWith()
+									.status(SUCCESS)
+									.contractCallResult(resultWith().gasUsed(10_000L)
+										.contractCallResult(functionResult()
+											.forFunction(FunctionResult.FunctionType.MINT)
+											.withStatus(SUCCESS)
+											.withTotalSupply(1L)
+											.withSerialNumbers(1L)
```

Comment:
Format this block so that resultWith() is on a new line and fix the indentation. (tabs)